lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -3014,12 +3044,12 @@ void PartitionCloner::handleSwitchEnum(SwitchEnumInst *inst) {
 /// Insert a send of values from the specified instruction result(s) to the
 /// accelerator, and insert receives in it.
 void PartitionCloner::insertSend(SILInstruction &inst) {
-  if (isa<TermInst>(inst)) {
-    if (auto *SEI = dyn_cast<SwitchEnumInst>(&inst)) {
-      handleSwitchEnum(SEI);
+  if (auto *Term = dyn_cast<TermInst>(&inst)) {
```

Comment:
nit: our local coding style is to start a var name with lower case -- "term", or we also do "TI"

Reply:
Fixed.